Feature/debug

kobizz opened the pull request.

joshmarom committed:
Merge branch 'release/1.2.1' of https://github.com/pojome/elementor into release/1.2.1

joshmarom committed:
Include Swiper and change image carousel

joshmarom committed:
Created a new widget to demonstrate swiper image carousel

joshmarom committed:
Merge branch 'release/1.2.2' of https://github.com/pojome/elementor into feature/swiper

joshmarom committed:
Merge branch 'release/1.2.2' of https://github.com/pojome/elementor into feature/swiper

joshmarom committed:
swiper image carousel

joshmarom committed:
Merge branch 'release/1.2.4' of https://github.com/pojome/elementor into feature/swiper

joshmarom committed:
Merge branch 'release/1.2.4' of https://github.com/pojome/elementor into feature/swiper

joshmarom committed:
swiper scss reduction and arrows like slick

joshmarom committed:
Merge branch 'release/1.2.4' of https://github.com/pojome/elementor into feature/swiper

matipojo committed:
debugger

KingYes committed:
Bump to 1.3.3

KingYes committed:
After release

KingYes committed:
Update readme

kobizz committed:
Merge branch 'release/1.3.3' of https://github.com/pojome/elementor into feature/error_log

# Conflicts:
#	assets/js/editor.min.js

kobizz committed:
Automatically convert lines break to `<br>` tag in html view

kobizz committed:
Allow to dynamically add reports

kobizz committed:
‘Debugger’ => ‘Debug’

kobizz committed:
Rewrite debug script

kobizz committed:
Improve debug functionality

kobizz committed:
Added modules folder

kobizz committed:
Adding custom error when multiple control value is invalid

kobizz committed:
Compiled files

kobizz committed:
Fixed error parsing for multi browsers

kobizz committed:
Use the elementor config ajax url

kobizz committed:
Compiled files

matipojo commented on includes/modules/debug/debug.php:
```diff
@@ -0,0 +1,92 @@
+		}
+
+		if ( self::MAX_LOGS_TO_SAVE && self::MAX_LOGS_TO_SAVE < count( $log ) ) {
+			$log = array_splice( $log, -10 );
```
@cobicarmel , Please change `10` to `self::MAX_LOGS_TO_SAVE`

matipojo commented on includes/modules/debug/assets/js/debug.js:
```diff
@@ -0,0 +1,112 @@
+	};
+
+	this.addCustomError = function( error ) {
+		var errorInfo = {
```
add a `type` argument to categorize the errors

matipojo commented on includes/modules/debug/assets/js/debug.js:
```diff
@@ -0,0 +1,112 @@
+			return;
+		}
+
+		self.addError( originalEvent.message, originalEvent.filename, originalEvent.lineno, originalEvent.colno );
```
add a `type` argument to categorize the errors

kobizz committed:
Merge branch 'release/1.3.3' of https://github.com/pojome/elementor into feature/debug

kobizz committed:
Use constant

kobizz committed:
this => self

kobizz committed:
Send more detailed info about the error

kobizz committed:
custom => customFields

kobizz committed:
Added ‘type’ parameter

kobizz committed:
Variable

kobizz committed:
Added ‘category’ and ‘tag’

kobizz committed:
Extending with defaults

kobizz committed:
Compiled files

matipojo commented on includes/modules/debug/assets/js/debug.js:
```diff
@@ -0,0 +1,128 @@
+	var initSettings = function() {
+		settings = {
+			debounceDelay: 500,
+			urlsToWatch: [
```
Please add a method to add a Watch Url:

    addWatchUrl: function( url ) {
        var urls = this.getSettings( 'urlsToWatch' );
        urls.push( url );
    },

KingYes committed:
Fix detect if all images are loaded

anandau committed:
Correction in  js hook's example -

Made a correction examples of  following js hook
frontend/element_ready/{elementType.skinName}

KingYes committed:
Make method as public

KingYes committed:
Merge pull request #1445 from anandau/master

Correction in  js hook's example in Docs

KingYes committed:
Merge branch 'master' into release/1.3.4

joshmarom committed:
Merge branch 'release/1.3.4' of https://github.com/pojome/elementor into feature/swiper

joshmarom committed:
Remove  image-swiper widget

KingYes committed:
Merge pull request #1451 from joshmarom/feature/swiper-assets

Feature/swiper assets

KingYes committed:
Revent last commit

KingYes committed:
Update packages

KingYes committed:
Added revisions for all Elementor CPTs

kobizz committed:
Shapes fix for RTL pages

KingYes committed:
Merge pull request #1457 from cobicarmel/release/1.3.4

Shapes fix for RTL pages

arielk committed:
Removed note for minify css

arielk committed:
Removed lightbox section 2

arielk committed:
Removed lightbox section 3

arielk committed:
Removed lightbox section 4

KingYes committed:
Merge pull request #1458 from arielk/release/1.3.4

Removed note for minify css

KingYes committed:
Fix revision option

KingYes committed:
Merge branch 'release/1.3.4' of github.com:pojome/elementor into release/1.3.4

kobizz committed:
Merge branch 'release/1.3.4' of https://github.com/pojome/elementor into feature/debug

# Conflicts:
#	assets/css/frontend-rtl.min.css
#	assets/css/frontend.min.css
#	assets/js/editor.min.js

kobizz committed:
Added `addURLToWatch` method
